requirements_all.txt

```diff
@@ -159,3 +159,6 @@ pushetta==1.0.15
 
 # Orvibo S10
 orvibo==1.0.0
+
+# Ecobee (*.ecobee)
+https://github.com/nkgilley/home-assistant-ecobee-api/archive/e0388659a0f2fc7266485affbd398350cc0b5c58.zip#python-ecobee==0.1.1
```

Comment:
Would you be able to rename the repository to not contain the name Home Assistant? That way, other people will easier realize that the file can be used without Home Assistant too.